### DISCUSSION TITLE
Search Results....

### DISCUSSION THREAD

**Discussion by chspans** [jagrosh/MusicBot#758] [Ideas]
Is there any way we can put a config option to be able to return more than 4 results when using the search function?

**Comment by Mattwmaster58** [comment]
This is easy to add, but exposes another flaw: the reaction system is simply to slow for this purpose, and that's why I suspect only 4 was used

**Reply by 2yysforu** [reply]
so you cant?

**Reply by Mattwmaster58** [reply]
no, you can't unless you edit the code or a new release is made.

**Comment by 2yysforu** [comment]
Thanks for the information
